### ISSUE TITLE
Boots with 20% damage taken on block + Glancing Blows = 115% damage taken

### ISSUE THREAD

**Issue by coldino** [PathOfBuildingCommunity/PathOfBuilding#2287]
A block build using Glancing Blows and the new boots with `20% damage taken on block` implicit leads to very unrealistic EHP due to incorrect chance to not take damage calculation, showing 115% damage taken from blocks:

Chance to not take damage when hit **without** boots:
![image](https://user-images.githubusercontent.com/28345893/109647873-c1dd3800-7b51-11eb-8f49-0165f4eab536.png)

Chance to not take damage when hit **with** boots:
![image](https://user-images.githubusercontent.com/28345893/109647771-a07c4c00-7b51-11eb-983d-be37f2261197.png)

Very simple repro build: https://pastebin.com/y9gGFztW

Note: I'm not sure how the boot effect should stack with GB, but it can't be 115% effective damage taken from blocks!

**Comment by pundm** [comment]
'This page is no longer available. It has either expired, been removed by its creator, or removed by one of the Pastebin staff.'

**Comment by coldino** [comment]
Damn, pastebin strikes again.

Well repro is real simple:
![image](https://user-images.githubusercontent.com/28345893/109654705-6a8f9580-7b5a-11eb-83f8-f4490c484ce7.png)

Shield:
```
Test Item
Pinnacle Tower Shield
Quality: 20
Sockets: R-R-R
LevelReq: 70
Implicits: 1
+30 to maximum Life
```

Boots:
```
Test Item
Basemetal Treads
Item Level: 21
Quality: 20
Sockets: R-R
LevelReq: 20
Implicits: 2
+5% Chance to Block Attack Damage
You take 20% of Damage from Blocked Hits
```

**Comment by pundm** [comment]
Fixed in 1.4.170.28
